torchbenchmark/util/extra_args.py

```diff
@@ -44,7 +44,9 @@ def check_precision(model: 'torchbenchmark.util.model.BenchmarkModel', precision
             return True
         if model.test == 'train' and model.device == 'cuda':
             return hasattr(model, 'enable_amp') or is_staged_train_test(model)
-    assert precision == "fp32", f"Expected precision to be one of fp32, tf32, fp16, or amp, but get {precision}"
+    if precision == "fx_int8":
+        return model.device == 'cpu' and hasattr(model, "enable_fx_int8")
+    assert precision == "fp32", f"Expected precision to be one of fp32, tf32, fp16, fx_int8, or amp, but get {precision}"
```

Comment:
We can create an array `AVAILABLE_PRECISIONS` and use it in this message and the choices of the `precision` option